pkg/util/sys/linux/sys_linux.go

```diff
@@ -19,9 +19,22 @@ import (
 	"net"
 	"syscall"
 
+	"golang.org/x/exp/constraints"
 	"golang.org/x/sys/unix"
 )
 
+func charsToString[T constraints.Integer](ca []T) string {
```

Comment:
Do we have an example when it will be uint8 instead of int8? Maybe we could add a ut.

Reply:
I do. But that example is compiled on riscv64-glibc, where it is indeed `uint8`. It is too platform-dependent. So I think it is ok that there is no tests.